06-files-input-and-output/high_scores.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+import csv
+import os
+
+def parse_csv(file):
```

Comment:
`file` is too general and might shadow a built in function of python (I'm not sure). Anyway I think `csv_file` will be better here.